gen/calc.py

```diff
@@ -288,9 +276,7 @@ def validate_dcos_overlay_network(dcos_overlay_network):
 
 
 def calculate_oauth_available(oauth_enabled):
```

Comment:
nit: this could just be `return oauth_enabled` now, optionally with a `validate_true_false`